apps/settings/js/panels/bluetooth/panel.js

```diff
@@ -88,8 +88,7 @@ define(function(require) {
             actionMenuDialog: panel.querySelector('.paired-device-option'),
             connectOption: panel.querySelector('.connect-option'),
             disconnectOption: panel.querySelector('.disconnect-option'),
-            unpairOption: panel.querySelector('.unpair-option'),
-            cancelOption: panel.querySelector('.cancel-option')
```

Comment:
might be https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/panels/bluetooth/panel.js#L488 ?